microsoft/testsuites/core/provisioning.py

```diff
@@ -191,6 +193,31 @@ def verify_reboot_in_platform(
             case_name="verify_reboot_in_platform",
         )
 
+    @TestCaseMetadata(
+        description="""
+        This case runs smoke test on a node provisioned with ultra disk.
+        The test steps are same as `smoke_test`.
+        """,
+        priority=3,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                data_disk_type=schema.DiskType.UltraSSDLRS,
+                os_disk_type=schema.DiskType.StandardHDDLRS,
```

Comment:
Do we need to place a requirement on `os_disk_type`? Maybe we should let it follow the default / let LISA decide.

Reply:
Removed it.